GitHubRepositories.ps1

```diff
@@ -818,19 +851,31 @@ function Update-GitHubRepository
         the background, enabling the command prompt to provide status information.
         If not supplied here, the DefaultNoStatus configuration property value will be used.
 
+    .OUTPUTS
+        GitHub.Repository
+
     .EXAMPLE
-        Update-GitHubRepository -OwnerName Microsoft -RepositoryName PowerShellForGitHub -Description 'The best way to automate your GitHub interactions'
+        Update-GitHubRepository -OwnerName microsoft -RepositoryName PowerShellForGitHub -Description 'The best way to automate your GitHub interactions'
 
         Changes the description of the specified repository.
 
     .EXAMPLE
         Update-GitHubRepository -Uri https://github.com/PowerShell/PowerShellForGitHub -Private:$false
 
         Changes the visibility of the specified repository to be public.
+
+    .EXAMPLE
+        Get-GitHubRepository -Uri https://github.com/PowerShell/PowerShellForGitHub |
+            Update-GitHubRepository -NewName 'PoShForGitHub' -Force
+
+        Renames the repository without any user confirmation prompting.  This is identical to using
+        Rename-GitHubRepository -Uri https://github.com/PowerShell/PowerShellForGitHub -NewName 'PoShForGitHub' -Confirm:$false
 #>
     [CmdletBinding(
         SupportsShouldProcess,
-        DefaultParameterSetName='Elements')]
+        DefaultParameterSetName='Elements',
+        ConfirmImpact='High')]
```

Comment:
Is `ConfirmImpact='High'` needed on a Set function?

Reply:
That only goes into effect _specifically_ when the user is choosing to rename the repo.  That _seems_ like a significant enough action to require confirmation (and is consistent with the behavior of `Rename-GitHubRepository`.

Reply:
Ok, but it will need removing and some refactoring when #225 is implemented for this function.